k8_cortx_cloud/deploy-cortx-cloud.sh

```diff
@@ -733,6 +759,35 @@ function deployCortxConfigMap()
         --from-file="${auto_gen_ha_path}"
 }
 
+# https://stackoverflow.com/a/26665585/508923
+function choose()
+{
+    echo "${1:RANDOM%${#1}:1}" "${RANDOM}";
```

Comment:
I'm not a security expert, but this might raise some red flags. `$RANDOM` is [poor source](https://tldp.org/LDP/abs/html/randomvar.html) of randomness, meaning the passwords will be low quality. Might be better to fetch from `/dev/urandom`, which I think works for both Linux and MacOS and is very secure. Lots of examples here: https://unix.stackexchange.com/questions/230673/how-to-generate-a-random-string

Reply:
You're right.  I have taken another shot at this now.